src/components/UI/Navigation/Navigation.js

```diff
@@ -72,6 +72,38 @@ const OuterNav = styled.nav`
   }
 `;
 
+const BreadcrumbWrapper = styled.div`
+  line-height: 20px;
+  background-color: ${({ theme }) => theme.colors.shade3};
+  padding: 7px;
+  border-bottom: ${({ theme }) => theme.colors.shade6} 1px solid;
+  border-top: ${({ theme }) => theme.colors.shade1} 1px solid;
+  font-size: 11px;
+  clear: both;
+
+  @media only screen and (max-width: 1024px) {
```

Comment:
Can we use the existing utility here?

https://github.com/giantswarm/happa/blob/7341cdfdc44a2319a16fbfa18a2f9f42518b4fe6/src/components/UI/AppDetails/AppDetails.js#L106